vpdq/python/vpdq.pyx

```diff
@@ -44,7 +43,7 @@ cdef extern from "vpdq/cpp/hashing/vpdqHashType.h" namespace "facebook::vpdq::ha
 
 
 cdef extern from "vpdq/cpp/hashing/filehasher.h" namespace "facebook::vpdq::hashing":
-    bool hashVideoFileFFMPEG(
+    bool hashVideoFile(
         string input_video_filename,
         vector[vpdqFeature]& pdqHashes,
         string ffmpeg_path,
```

Comment:
I'm planning on removing the FFmpeg path residuals from CPP and then Python after this PR is merged. I want to ask here if for backwards compatibility for both the CPP CLI and Python function computeHash() the parameters should remain and do nothing, or if they can be removed which might cause errors.